docs/src/basic_usage.md

```diff
@@ -0,0 +1,27 @@
+# Basic usage
+
+You can calculate Riemannian gradient of a function defined in its embedding in multiple ways.
+For example, `rb_onb_fd51` corresponds to a finite differencing scheme and `rb_onb_fwdd` calculates gradient using `ForwardDiff.jl`. [`DifferentiationInterface.jl`](https://github.com/JuliaDiff/DifferentiationInterface.jl) is used to select the backend.
```

Comment:
could we link `rb_onb_fd51`, `rb_onb_fwdd`, and `ForwardDiff` as well?

Reply:
`rb` things are defined in the code below so I'm not sure how to link that? I can link ForwardDiff though.

Reply:
I'm not sure how to succinctly express this beyond what is already there. What piece of information about these variables do you consider important for this introduction?

Reply:
I was only a bit confused they are mentioned before defined/introduced, hence the “the following”.

That is a bit like starting to speak about Frank, but only a few sentences later telling which Frank.

Reply:
I've moved things around, I hope it's better now.